### PR TITLE
Update UG Features

### PR DESCRIPTION
**Pull request by joenzkimchan**
Features currently are for AB3, lets change them to fit the feature set of v1.2 of StudyTracker.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -64,96 +64,152 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
 ![help message](images/helpMessage.png)
 
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a study spot: `add`
 
 Adds a person to the address book.
```
Comment (mslevis) [inline docs/UserGuide.md:82]: can change this to study spot too

**Review by mslevis — CHANGES_REQUESTED** [review]
Great work! just some small spots you missed out.

**Comment by joenzkimchan** [comment]
Have made the changes, do take a look again!

**Review by mslevis — APPROVED** [review]
LGTM!

**Pull request merged**